spaces/S000068/properties/P000038.md

```diff
@@ -7,5 +7,4 @@ refs:
   name: Counterexamples in Topology
 ---
 
-Asserted in the General Reference Chart for space #76 in
-{{doi:10.1007/978-1-4612-6290-9_6}}.
+Every Euclidean circle with subset topology is homeomorphic to {S170} which is {P38}. Any two points lie on a common circle.
```

Comment:
```suggestion
Every Euclidean circle with the subspace topology induced from $X$ is homeomorphic to {S170} which is {P38}. Any two points lie on a common circle.
```
clarify to avoid confusion with the subspace topology induced from the Euclidean topology (even if it does not really matter after the fact)